src/modules/datepicker/directives/datepicker.directive.ts

```diff
@@ -60,6 +74,9 @@ export class SuiDatepickerDirective
                 this.config = new TimeConfig();
                 break;
         }
+        if (this._initialView != undefined) {
```

Comment:
Thoughts on making `initialView?:CalendarViewType` a constructor parameter of `CalendarMapping`?